python/torch_mlir/extras/fx_importer.py

```diff
@@ -135,7 +135,6 @@
     func as func_dialect,
 )
 
-from .._mlir_libs._torchMlir import get_int64_max, get_int64_min
```

Comment:
Oh, these functions can also be removed from `python/TorchMLIRModule.cpp` (they were only added there to be used from this file)